crates/ika-core/src/dwallet_mpc/mpc_session.rs

```diff
@@ -523,29 +471,10 @@ impl DWalletMPCSession {
     pub(crate) fn check_quorum_for_next_crypto_round(&mut self) -> ReadyToAdvanceCheckResult {
```

Comment:
```suggestion
    pub(crate) fn check_quorum_for_next_crypto_round(&self) -> ReadyToAdvanceCheckResult {
```

I think this should work now